dynamiqs/result.py

```diff
@@ -158,7 +158,7 @@ class MEResult(Result):
         - If the option `cartesian_batching = True` (default value), the results leading
           dimensions are
           ```
-          ... = ...H, ...L0, ...L1, (...), ...rho0
+          ... = ...H, ...L0, ...L1, ...Ln, ...rho0
```

Comment:
I would revert this, it's incorrect.

Reply:
(`n` is typically used for the Hilbert space dimension, and this does not clearly explain that there are arbitrarily many jump operators)

Reply:
Ok, I didn't really like the previous notation, but I agree this one is not good. I'm reverting for lack of something better.